x-pack/solutions/security/plugins/security_solution/public/entity_analytics/hooks/use_criticality_level_colors.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { useEuiTheme } from '@elastic/eui';
+import type { CriticalityLevelWithUnassigned } from '../../../common/entity_analytics/asset_criticality/types';
+// TODO: update these colors once severity color palette is available
+export const useCriticalityLevelColors = (): Record<CriticalityLevelWithUnassigned, string> => {
```

Comment:
I see that @angorayc is doing a similar work [here](https://github.com/elastic/kibana/pull/204631/files#diff-7de6dd565ec60da487f83a894070d2fbe59f3804694850b349d6b731f714262aR18) so I wonder if there would be a value in extracting this logic into a more generic function/hook that could be moved into our `common` folder.
Also you can see on her PR that there is a discussion regarding colors that are changing/have changed.

Reply:
Thanks, will have a look! Agreed, if its the same colors and mappings then 100%.